catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswSourceConfiguration.java

```diff
@@ -92,7 +103,11 @@ public String getPassword() {
     }
 
     public void setPassword(String password) {
```

Comment:
I'm not a fan of passing back nulls from any sort of API with some caveats. In my opinion, we should pass back from the method whatever it receives. If we pass in an unencrypted password, we should pass back that password. Empty string is an unencrypted password and we should just pass that back as well. Same also goes for null. If null is passed in, we should probably just hand null right back.